webxray/src/mix-master.js

```diff
@@ -204,7 +219,7 @@
                     // The dialog may have decided to replace all our spaces
                     // with non-breaking ones, so we'll undo that.
                     var html = data.endHTML.replace(/\u00a0/g, " ");
-                    var newContent = self.replaceElement(focusedElement, html);
+                    var newContent = self.replaceElement(originalElement, focusedElement, html, data.finished);
```

Comment:
On the next line, `focusedElement` changes in this scope every time the `ok` message comes through from the editor dialog. I want to capture the element before the user gets a chance to change it.